docs/pages/tutorial/eas/internal-distribution-builds.mdx

```diff
@@ -0,0 +1,191 @@
+description: Learn about what are internal distribution builds, why we need them, and how to create them.
+---
+
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+import { CODE } from '~/ui/components/Text';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+In this chapter, we're going to learn how to set up [internal distribution builds](/guides/sharing-preview-releases/#internal-distribution). Creating an internal build is quite similar to creating a development build. However, the difference is how it is used and distributed.
+
+## Internal distribution build
+
+An internal distribution build is great for sharing an app's progress with our team (technical and non-technical stakeholders) to gather feedback without running a development server. Our team can directly install the build and run it like an app on their device.
+
+## Ways to distribute an app internally
+
+### Traditional methods
+
+- **Android**: Using Google Play beta
+- **iOS**: Using TestFlight
+
+However, both of these traditional methods have their limitations. For example, TestFlight limits to one active build at a time.
+
+### EAS Build for faster distribution
+
+EAS Build speeds up the process. It creates shareable links for our builds and provides instructions on using them. It has a default configuration designed to facilitate internal distribution, offering a more efficient alternative to traditional methods.
+
+## Steps to create an internal distribution build
+
+To create and distribute a build with EAS Build, we need to:
+
+- Configure a build profile
+- Add test devices for iOS to a provisioning profile
+- Trigger a build for Android and iOS and install them on devices
+
+<Step label="1">
+
+## Configure a build a profile
+
+In the previous chapter, we learned that after initializing **eas.json** in our project, it already contains some default configuration. In this configuration, there is a pre-configured build profile called `preview`:
+
+```json eas.json
+{
+  "build": {
+    "preview": {
+      "distribution": "internal"
+    }
+  }
+}
+```
+
+This is all we need to create our first internal distribution build. The `preview` build profile from the above snippet has a `distribution` property whose value is set to `internal`. This value allows us to share our build URLs with anyone so they to install it on their device and do not require a development server to run the app.
+
+As discussed in the previous chapters, for non-app store builds, Android requires **.apk** and iOS needs **.ipa** formats. This applies to internal distribution builds as well. The `distribution` when set to `internal`, automatically creates the app binary in these file formats.
```

Comment:
"The distribution when set to internal, automatically creates the app binary in these file formats." -> only for device builds. On a simulator it builds an `app` as you said.

Reply:
Add the context, thank you for catching this!